metrics/tarantool/config.lua

```diff
@@ -0,0 +1,76 @@
+local utils = require('metrics.utils')
+
+local collectors_list = {}
+
+local function get_config_alerts(config_info)
+    -- https://github.com/tarantool/tarantool/blob/319357d5973d15d08b8eda6a230eada08b710802/src/box/lua/config/utils/aboard.lua#L17-L18
+    local config_alerts = {
+        warn = 0,
+        error = 0,
+    }
+
+    for _, alert in pairs(config_info.alerts) do
+        config_alerts[alert.type] = config_alerts[alert.type] + 1
+    end
+
+    return config_alerts
+end
+
+local function get_config_status(config_info)
+    -- See state diagram here
+    -- https://github.com/tarantool/doc/issues/3544#issuecomment-1866033480
+    local config_status = {
+        uninitialized = 0,
+        startup_in_progress = 0,
+        reload_in_progress = 0,
+        check_warnings = 0,
+        check_errors = 0,
+        ready = 0,
+    }
+
+    config_status[config_info.status] = 1
```

Comment:
Maybe it's better to choose a unique numerical value for each status instead of passing the status name to labels?

Reply:
Such approach is bad at scaling: in case some new status will be introduced, we will have two ways:
- add a value in-between: very bad, will break existing dashboards;
- add a value to the tail: is bad in case it's not one more "healthy" status (which is unlikely), since healthy "ready" status will always be visualized as something between "reload_in_progress" and "new_not_healthy_status".

Such metrics are also impossible to read without documentation. Similar approach is already discussed in [1, 2]. So I think this approach is a bit better, even though it's harder to visualize.
1. https://github.com/prometheus/client_python/issues/416
2. https://github.com/open-telemetry/opentelemetry-specification/issues/1711